exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,6 @@
 // Declare your function here
 
-const greeting = createLongGreeting("Daniel", 30);
-
-console.log(greeting);
+function createLongGreeting(name, age) {
```

Comment:
Modifying the code from exercise3.js in this folder, can you modify createLongGreeting to return a greeting with both the name ( a string ) and an age ( a number ) ?